OloEditor/src/Panels/ContentBrowserPanel.cpp

```
@@ -99,27 +296,237 @@ namespace OloEngine
+
+                    OLO_CORE_INFO("Created material: {}", matPath.string());
+                    ImGui::CloseCurrentPopup();
+                }
+                ImGui::EndMenu();
+            }
+
+            ImGui::EndPopup();
+        }
+    }
+
+    void ContentBrowserPanel::DrawFileContextMenu(const std::filesystem::path& path, ContentFileType fileType)
```

Clang (14.0.6)

[warning] method 'DrawFileContextMenu' can be made static

(readability-convert-member-functions-to-static)